mlflow/pipelines/regression/v1/pipeline.py

```diff
@@ -99,6 +104,7 @@
 .. |'data' section in pipeline.yaml| replace:: `'data' section in pipeline.yaml <https://github.com/mlflow/mlp-regression-template/blob/35f6f32c7a89dc655fbcfcf731cc1da4685a8ebb/pipeline.yaml#L15-L32>`__
 .. |'metrics' section of pipeline.yaml| replace:: `'metrics' section of pipeline.yaml <https://github.com/mlflow/mlp-regression-template/blob/35f6f32c7a89dc655fbcfcf731cc1da4685a8ebb/pipeline.yaml#L64-L73>`__
 .. |'validation_criteria' section of the 'evaluate' step definition in pipeline.yaml| replace:: `'validation_criteria' section of the 'evaluate' step definition in pipeline.yaml <https://github.com/mlflow/mlp-regression-template/blob/35f6f32c7a89dc655fbcfcf731cc1da4685a8ebb/pipeline.yaml#L47-L56>`__
+.. |'tuning' section of the 'train' step definition in pipeline.yaml| replace:: `'tuning' section of the 'train' step definition in pipeline.yaml <https://github.com/mlflow/mlp-regression-template/blob/76f3ed1a12decd3f5c15a50b5f9ef974eb50a94b/pipeline.yaml#L56-L70>`__
```

Comment:
Can we add an inline comment documenting `early_stop_fn` and also add a discrete parameter like `penalty: values: ....`?

Reply:
Can we also set `parallelism` to 1 by default so that users don't need to have PySpark to run the example as-is. Can we also document that `max_iters` should be increased when `parallelism > 1`?

Reply:
Can we make `steps.train.my_early_stop_fn` refer to a real early stopping function, e.g. by supplying a hyperopt builtin function instead?

Reply:
Can we document the `estimator_params` section using an inline comment?

Reply:
Can we document `sample_fraction` by adding an inline comment?